content/docs/state-and-lifecycle.md

```diff
@@ -68,25 +68,25 @@ ReactDOM.render(
 );
 ```
 
-To implement this, we need to add "state" to the `Clock` component.
+Para implementar esto, necesitamos agregar «estado» al componente `Clock`.
```

Comment:
No biggie, but I'm curious, why did you write `«estado»` instead of `"estado"`?

Reply:
Actually, ["Spanish typography"](https://es.wikipedia.org/wiki/Comillas), the problem is that now I see this is not the norm everywhere. In Spain "Spanish" quotes are the norm but in some places like Mexico due to US influence (and also technology I guess), the most common are "English" quotes. In Cuba, for instance, Spanish quotes are used in books and (media)[http://www.granma.cu/deportes/2019-01-31/la-serie-del-caribe-en-cuba-seria-un-bonito-espectaculo-31-01-2019-21-01-20].
Any way can work I think. Once we arrive at a decision we should put in on the style guide and then make the appropriate changes in all the pages.